Lib/idlelib/editor.py

```diff
@@ -8,6 +8,7 @@
 import tokenize
 import traceback
 import webbrowser
+import winreg
```

Comment:
Do you need to put this somewhere in a Windows-only codepath? It won't exist on other platforms

Reply:
Yes.  Moved to the patched Windows-only code.

Reply:
Yes, I moved it to the Windows-only, run-once code.